resources/wikia/modules/spec/integration/loader.spec.js

```diff
@@ -112,37 +111,37 @@ describe("Loader Module", function () {
 			loader = modules['wikia.loader'](windowMock, mwMock, nirvanaMock, jQuery, logMock);
 
 		// check calls to this function
-		spyOn(mwMock.loader, 'use').andCallThrough();
+		spyOn(mwMock.loader, 'use').and.callThrough();
 
 		loader({
 			type: loader.LIBRARY,
 			resources: ['mustache']
 		}).
-			done(function() {
-				expect(mwMock.loader.use).toHaveBeenCalled();
-				done();
-			});
+		done(function () {
```

Comment:
The goal of this task was to update JS unit test environment. I had to fix error caused by Jasmine API change. Before committing changes I did run JS beautify on these files and they all pass JSCS and JSHint (I'm using our pre-commit hooks).